modules/serverless-creating-subscriptions-kn.adoc

```diff
@@ -25,7 +25,7 @@ $ kn subscription create <subscription_name> \
   --sink <sink_prefix>:<sink_name> \ <2>
   --sink-dead-letter <sink_prefix>:<sink_name> <3>
 ----
-<1> `--channel` specifies the source for cloud events that should be processed. You must provide the channel name. If you are not using the default `InMemoryChannel` channel that is backed by the `Channel` custom resource, you must prefix the channel name with the `<group:version:kind>` for the specified channel type. For example, this will be `messaging.knative.dev:v1beta1:KafkaChannel` for an Apache Kafka backed channel.
+<1> `--channel` specifies the source for cloud events that should be processed. You must provide the channel name. If you are not using the default `InMemoryChannel` channel that is backed by the `Channel` custom resource, you must prefix the channel name with the `<group:version:kind>` for the specified channel type. For example, this will be `messaging.knative.dev:v1:KafkaChannel` for an Apache Kafka backed channel.
```

Comment:
While `Channel` would be `messaging.knative.dev:v1:Channel` , the `KafkaChannel` is still just `v1beta1`, so this line should stay as it was.